src/test/java/com/datadog/api/TestUtils.java

```diff
@@ -177,6 +177,11 @@ public static void getSecretsFromEnv() {
 
         @Before
         public void setupMockServer() {
+            // Mockserver uses a connection pool with keepAlive connections to talk to the API.
```

Comment:
👍 🙇 🦸

Reply:
Can we have a single `mockServer` (per thread) instead creating and removing one for each test case?

Reply:
That'd be a good way to make things faster I believe. We just need to make sure we still make one cassette per test though.
IIRC I had to go this way with one mock server instance per test for this reason. There might be a better way though.

Reply:
I do want to work on that, but not in this PR (FTR, the connection reusing is done even within a single testcase when there are multiple calls to the API).